physics/eff-mass.cpp

```diff
@@ -0,0 +1,116 @@
+#include <LatAnalyze/Core/OptParser.hpp>
```

Comment:
this file is too specific to go into LatAnalyze, please move it to your analysis code